[2024-05-19T20:14:11.580+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-19T20:14:07.207443+00:00 [queued]>
[2024-05-19T20:14:11.591+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-19T20:14:07.207443+00:00 [queued]>
[2024-05-19T20:14:11.591+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-05-19T20:14:11.612+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): seceond_task> on 2024-05-19 20:14:07.207443+00:00
[2024-05-19T20:14:11.622+0000] {standard_task_runner.py:60} INFO - Started process 93 to run task
[2024-05-19T20:14:11.626+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'seceond_task', 'manual__2024-05-19T20:14:07.207443+00:00', '--job-id', '394', '--raw', '--subdir', 'DAGS_FOLDER/frist-dag.py', '--cfg-path', '/tmp/tmp7zmhjm3b']
[2024-05-19T20:14:11.627+0000] {standard_task_runner.py:88} INFO - Job 394: Subtask seceond_task
[2024-05-19T20:14:11.693+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-19T20:14:07.207443+00:00 [running]> on host 7b23a6949395
[2024-05-19T20:14:11.787+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='seceond_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-19T20:14:07.207443+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-19T20:14:07.207443+00:00'
[2024-05-19T20:14:11.789+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-19T20:14:11.790+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'wc -l /opt/***/output.csv']
[2024-05-19T20:14:11.803+0000] {subprocess.py:86} INFO - Output:
[2024-05-19T20:14:11.813+0000] {subprocess.py:93} INFO - 240019 /opt/***/output.csv
[2024-05-19T20:14:11.814+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-19T20:14:11.837+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=seceond_task, execution_date=20240519T201407, start_date=20240519T201411, end_date=20240519T201411
[2024-05-19T20:14:11.878+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-19T20:14:11.897+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
